app.go

```diff
@@ -240,6 +246,31 @@ func executeQuote(cmd command) bool {
 		return false
 	}
 
+	//Check auto buy sell
+
+	for _, v := range (*autoBuyRequestStore)[stock] {
```

Comment:
If the quote isn't fresh it will get refreshed. getQuote is guaranteed to return a fresh-to-death quote.